### ISSUE TITLE
App title

### ISSUE THREAD

**Issue by wlla** [NYPL/staff-picks#67]
App `title` should be "Best Books for Kids" and "Best Books for Teens" respectively. Currently is "NYPL | Recommendations".

**Comment by rhernand3z** [comment]
I assume this will be configurable and based on the route to determine `Kids` or `Teens`.

**Comment by EdwinGuzman** [comment]
Yes, that's where the `react-doc-meta` plugin came in handy.

**Comment by rhernand3z** [comment]
Seems like this can easily be handled on the server though, don't see a need for an outdated plugin dependency when our app routes are very specific...
